client/my-sites/post-type-filter/index.jsx

```diff
@@ -205,11 +214,7 @@ export class PostTypeFilter extends Component {
 							fitsContainer
 							initialValue={ query.search }
 							onSearch={ this.props.doSearch }
-							placeholder={ this.props.translate( 'Search %(postTypes)s…', {
-								args: {
-									postTypes: typeLabel,
-								},
-							} ) }
+							placeholder={ `${ searchPagesPlaceholder }...` }
```

Comment:
Same as above:
```suggestion
							placeholder={ `${ searchPagesPlaceholder }…` }
```

Reply:
Damn... My IDE was replacing them last time I checked. Not sure why it didn't this time... I'll fix it.